core/domain/exp_services.py

```diff
@@ -792,6 +793,15 @@ def _save_exploration(committer_id, exploration, commit_message, change_list):
                 'which is too old. Please reload the page and try again.'
                 % (exploration_model.version, exploration.version))
 
+    if feconf.ENABLE_STRING_CLASSIFIER:
```

Comment:
@seanlip Wow. I need to apologise here. I used to use the version when I was working with CommitLogs, but since we changed to using change_lists, it was unnecessary. I'm really sorry about this, I'll make the changes